### DISCUSSION TITLE
Change folder directory

### DISCUSSION THREAD

**Discussion by annas01** [prasathmani/tinyfilemanager#726] [Q&A]
i have folder on /export/Emsada (example) and its outside root_path, how to show /export/Emsada from outside root_path?
i hv tried to change the root_path but it's not working and when im symlink /export/Emsada folder to /var/www/html it's show that folder but it's like the web freeze then i remove the symlink its normal again. can somebody help me? thanks before.

**Comment by prasathmani** [comment]
file manager will work only if file and folders are under the web root directory only.
